src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -113,6 +114,7 @@ public static void assertCommandFailure(Command command, Model actualModel, Stri
         assertEquals(expectedTutorsPet, actualModel.getTutorsPet());
         assertEquals(expectedFilteredList, actualModel.getFilteredStudentList());
     }
+
     /**
      * Updates {@code model}'s filtered list to show only the student at the given {@code targetIndex} in the
```

Comment:
Should the filtered list be specified?
```suggestion
     * Updates {@code model}'s {@code filteredStudents} list to show only the student at the given {@code targetIndex} in the
```